### PR TITLE
A few changes from last Huntsman observing run

### PR DESCRIPTION
**Pull request by wtgee**
* Make sure observation is still in valid list.
* Link pretty image from FITS to latest
* Weather plot fixes

### CODE REVIEW

File: pocs/scheduler/dispatch.py [hunk 1]
```diff
@@ -98,8 +98,9 @@ def get_observation(self, time=None, show_all=False, reread_fields_file=False):
             if self.current_observation is not None:
                 # Favor the current observation if still available
                 end_of_next_set = time + self.current_observation.set_duration
-                if end_of_next_set < common_properties['end_of_night'] and \
-                        self.observation_available(self.current_observation, end_of_next_set):
+                if (end_of_next_set < common_properties['end_of_night'] and
```
Comment (jamessynge) [inline pocs/scheduler/dispatch.py]: I'm struggling to decipher this expression. Please add more detail to the preceding comment.

File: pocs/utils/images/__init__.py [hunk 2]
```diff
@@ -90,7 +90,18 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
     if fname.endswith('.cr2'):
         return _make_pretty_from_cr2(fname, timeout=timeout, **kwargs)
     elif fname.endswith('.fits'):
-        return _make_pretty_from_fits(fname, **kwargs)
+        new_fname = _make_pretty_from_fits(fname, **kwargs)
+
+        try:
```
Comment (jamessynge) [inline pocs/utils/images/__init__.py]: Please add a comment such as:
```
# Symlink latest.jpg to the image; first remove the symlink if it already exists.
```

File: scripts/plot_weather.py [hunk 3]
```diff
@@ -696,7 +696,8 @@ def save_plot(self, plot_filename=None):
                 '$PANDIR'), 'weather_plots', plot_filename)
 
         print('Saving Figure: {}'.format(plot_filename))
-        self.fig.savefig(plot_filename, dpi=self.dpi, bbox_inches='tight', pad_inches=0.10)
+        self.fig.tight_layout()
```
Comment (jamessynge) [inline scripts/plot_weather.py]: I don't know what this is doing.

**Comment by wtgee** [comment]
I had added two new things from the latest of Huntsman runs:

* Option to post `say` messages into a slackbot channel. Requires a url in the config file to activate, does nothing by default
* Ability to set `solve-field` via an environment variable with a default of the current hard-coded location.

**Comment by wtgee** [comment]
Closing in favor of issues listed above.

**Pull request closed without merging**